lib/mountain_view/engine.rb

```diff
@@ -6,15 +6,21 @@ module MountainView
   class Engine < ::Rails::Engine
     isolate_namespace MountainView
 
+    initializer 'mountain_view.components_path' do |app|
+      MountainView.configure do |c|
+        c.components_path ||= app.root.join('app', 'components')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.